release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -321,6 +321,12 @@ For more information on configuration drift, see xref:../post_installation_confi
 
 You can now enable link:https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html[Linux control groups version 2] (cgroups v2) on specific nodes in your cluster. The {product-title} process for enabling cgroups v2 disables all cgroups version 1 controllers and hierarchies. The {product-title} cgroups version 2 feature is in Developer Preview and is not supported by Red Hat at this time.
 
+[id="ocp-4-10-poison-pill-operator"]
+==== Poison Pill Operator enhancements
+
+The Poison Pill Operator uses `NodeDeletion` as its default remediation strategy, which removes the `node` object. In {product-title} 4.10, the Poison Pill Operator introduces a new remediation strategy called `ResourceDeletion`, which removes the pods and associated volume attachments on the node rather than the `node` object. This strategy helps to recover workloads faster. 
```

Comment:
For readability, I'd break the first sentence into two.

I'd break up the second sentence, too.